bears/python/PyFlakesBear.py

```diff
@@ -0,0 +1,29 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable='pyflakes',
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>.*):\s*'
+                     r'(?P<severity>)\s(?P<message>.*)',
+        severity_map={
+            '': RESULT_SEVERITY.INFO
+        })
+class PyFlakesBear:
+    """
+    Checks Python files for logistic errors using ``pyflakes``.
+
+    See https://github.com/PyCQA/pyflakes for more info.
+    """
+    LANGUAGES = {"Python", "Python 3"}
+    REQUIREMENTS = {PipRequirement('pyflakes', '1.2.*')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/92503'
+    CAN_DETECT = {'Unused imports', 'Undefined Element'}
```

Comment:
@sils Well its with the regex. pyflakes gives a different output while checking Syntax errors -> https://asciinema.org/a/92541. May be I am bad with regex. I will update the PR.